xnat_tools/bids_utils.py

```diff
@@ -407,9 +407,24 @@ def assign_bids_name(
                 See documentation to understand behavior for repeated sequences."
             )
 
-        filesURL = host + "/data/experiments/%s/scans/%s/resources/DICOM/files" % (
+        # check the label to figure out which folder xnat has the dicoms stored
+        # in (DICOMS or secondary)
+        resp = get(
+            connection,
+            host + "/data/experiments/%s/scans/%s/resources" % (session, scanid),
```

Comment:
This URL is repeated a few times in this code segment.  We can minimize the repetition by specifying:
``` resourcesURL = f"/data/experiments/{session}/scans/{scanid}/resources" ``` and used respectively.